exchanges/okx/okx_types.go

```diff
@@ -314,9 +314,9 @@ type Instrument struct {
 
 // DeliveryHistoryDetail holds instrument id and delivery price information detail
 type DeliveryHistoryDetail struct {
-	Type          string  `json:"type"`
-	InstrumentID  string  `json:"insId"`
-	DeliveryPrice float64 `json:"px,string"`
+	Type          string                  `json:"type"`
+	InstrumentID  string                  `json:"insId"`
```

Comment:
Add this line to exclude_lines.txt so this linter can be skipped. The includes the tab space as well.

Reply:
done